actors/states/tree_top.go

```diff
@@ -92,3 +92,15 @@ func (t *TreeTop) LookupID(addr address.Address) (address.Address, error) {
 
 	return a, nil
 }
+
+func (t *TreeTop) ForEach(ctx context.Context, fn func(addr address.Address, actor *Actor) error) error {
+	var val Actor
+	return t.m.ForEach(&val, func(key string) error {
+		addr, err := address.NewFromString(key)
```

Comment:
@ZenGround0 this isn't the right converstion. We should put something with the new go-state-types keys that does it right (`NewFromBytes`)

Reply:
Resolved in #1147